extensions/binding/org.eclipse.smarthome.binding.lirc/src/main/java/org/eclipse/smarthome/binding/lirc/handler/LIRCRemoteHandler.java

```diff
@@ -59,9 +58,7 @@ public void handleCommand(ChannelUID channelUID, Command command) {
             return;
         }
         if (channelUID.getId().equals(LIRCBindingConstants.CHANNEL_TRANSMIT)) {
-            if (command instanceof RefreshType) {
-                // not supported
```

Comment:
Please keep this comment, otherwise other developers might wonder why the refresh command is not handled although every handler has to handle it.